.eslintrc.js

```diff
@@ -53,8 +53,8 @@ module.exports = {
     "no-debugger": "warn",
     "react/prop-types": "error",
     "curly": "error",
-    "semi": ["error", "always", {"omitLastInOneLineBlock": true}],
-    "semi-style": ["error", "last"],
+    // "semi": ["error", "always", {"omitLastInOneLineBlock": true}],
```

Comment:
Should this be commented out?

Reply:
What settings should we use, since much of the new code does not have semi-colons?

Reply:
either option is fine to me, semi-colons aren't really necessary in javascript